coderd/database/migrations/000023_site_config.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS site_config (
+    key varchar(256) NOT NULL UNIQUE,
+    value varchar(8192) NOT NULL
```

Comment:
Using `jsonb` as the value would make it much easier in the future to do migrations on structured data. We've needed to do this a couple times in v1.